bucket/retdec.json

```diff
@@ -0,0 +1,60 @@
+{
+    "version": "4.0",
+    "description": "A retargetable machine-code decompiler based on LLVM.",
+    "homepage": "https://github.com/avast/retdec",
+    "license": "MIT",
+    "notes": [
+        "The python executables were replaced with *.cmd by the shim creation.",

```

Comment:
```suggestion
```

This note is not required (it's implied)

Reply:
The user needs to know how to call the wrapper